Makefile.archlinux

```diff
@@ -143,23 +151,53 @@ dist-copy-out:
 
 ### Additional targets
 
+# Sign packages
+sign: sign_client = $(if $(GNUPG),$(GNUPG),gpg)
+sign:
+	@if [ -d $(ORIG_SRC)/$(OUTPUT_DIR) ]; then \
+		cd $(ORIG_SRC)/$(OUTPUT_DIR); \
+		for filename in *.pkg.tar.zst; do\
+			echo $$filename; \
+			$(sign_client) --yes --local-user "$(SIGN_ARCHLINUX_KEY)" --detach-sign -o "$$filename.sig" "$$filename";\
```

Comment:
SIGN_ARCHLINUX_KEY is inconsistent with other similar variables - we have plain SIGN_KEY and DEBIAN_SIGN_KEY.

Reply:
Yes indeed. I mismatched it when writing it.